tox.ini

```diff
@@ -76,6 +76,7 @@ deps =
     {[testenv]deps}
     behave==1.2.4
     behave-pytest==0.1.1
+    /root/mesos.native-0.28.2-py27-none-any.whl
```

Comment:
Yeah, this is a weird one. Fortunately, because we're building our .deb inside a docker container, we can do this one crazy trick (python purists hate him!)

I'll add some comments explaining it.